app/src/main/java/de/westnordost/streetcomplete/quests/accepts_cards/AddAcceptsCards.kt

```diff
@@ -0,0 +1,45 @@
+package de.westnordost.streetcomplete.quests.accepts_cards
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.osmquests.Tags
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.IS_SHOP_OR_DISUSED_SHOP_EXPRESSION
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddAcceptsCards : OsmFilterQuestType<CardAcceptance>() {
+
+    override val elementFilter = """
+        nodes with (
+          amenity=restaurant
+          or amenity=fast_food
+          or amenity=cafe
+          or amenity=ice_cream
```

Comment:
```suggestion
          amenity ~ restaurant|fast_food|cafe|ice_cream
```
Can be shorter like this.

Also, @matkoniecz no space between key and value is allowed, but for consistency, please always add spaces as for other quests, it is formatted that way also.